### ISSUE TITLE
TFC addon suddenly stops working.

### ISSUE THREAD

**Issue by lexisman** [cmik/plugin.video.tfctv#13]
I think tfc just did some update on their site. This is the error log:
https://paste.kodi.tv/puqoguvole
Both windows and android iptv boxes have stop working. Thank you.

**Comment by cmik** [comment]
Hi @lexisman,
Issue has been identified and fixed.
New version 2.0.2 is available.
Thanks in advance for your feedbacks

**Comment by lexisman** [comment]
Thanks, it seems it is working, one thing I notice is that in the menu there is also the GMA Pinoy TV.

**Comment by cmik** [comment]
Thanks @lexisman 
Yes, it seems that TFC has recently started to add GMA content to iWantTFC like some series and live channels. I don’t know when but I’ve noticed it in their last newsletter.

**Comment by Jranns** [comment]
Thx for the update,but seems to take ages to load.

**Comment by cmik** [comment]
@Jranns, you can go to “Install from repository > Cmik Repo > Add-on repository > Cmik Repo > Check for updates”
Then go back to “Cmik Repo” and select “Video add-on > iWantTFC > Versions” then check if you see the 2.0.2 and force the update

**Comment by Jranns** [comment]
See attached image.

**Comment by Jranns** [comment]
Is updated see attached image.  Takes ages to load. A episode.

**Comment by Jranns** [comment]
![20230512_203234](https://github.com/cmik/plugin.video.tfctv/assets/26060012/091896e4-af8d-4d6d-936c-b2353c304c0c)

![20230512_203142](https://github.com/cmik/plugin.video.tfctv/assets/26060012/944e03c1-9fdc-4c7c-bdb1-52feb5c81e42)

**Comment by cmik** [comment]
@Jranns , do you mean when you play an episode?
no problem on my side when displaying episodes list and playback.

**Comment by Jranns** [comment]
Yes, it keeps loading. Takes over 5 minutes or more to bring up the episodes and even longer time to start the episode.  Also the repository doesnt open ,like the attachment i showed you before.

**Comment by Jranns** [comment]
Ive cleared kodis cache. Seems faster ill do some more testing thx.
